### PR TITLE
print-dot: Removed primary path prefix from graph labels

### PR DESCRIPTION
**Pull request by RenskeW**
Removed primary file path prefix from graph labels

### CODE REVIEW

**Comment by RenskeW** [comment]
Made some changes in test_print_dot() so expected output now corresponds to computed output.

File: tests/test_examples.py [hunk 1]
```diff
@@ -1036,38 +1035,37 @@ def test_print_dot() -> None:
...
     assert main(["--debug", "--print-dot", cwl_path], stdout=stdout) == 0
     computed_dot = pydot.graph_from_dot_data(stdout.getvalue())[0]
     computed_edges = sorted(
-        (urlparse(source).fragment, urlparse(target).fragment)
+        (source, target)
         for source, target in computed_dot.obj_dict["edges"]
     )
     expected_edges = sorted(
-        (urlparse(source).fragment, urlparse(target).fragment)
+        (source, target)
```
Comment (mr-c) [inline tests/test_examples.py:1063]:
```suggestion
        (source, target)
```

This cause the CI tests to fail. Fixable via `make format`

**Review by mr-c — CHANGES_REQUESTED** [review]
Please run `make format` :-)

**Comment by RenskeW** [comment]
Thanks! 😃

**Review by mr-c — APPROVED** [review]

**Pull request merged**